### ISSUE TITLE
Symmetric padding in pytorch

### ISSUE THREAD

**Issue by bsun0802** [VainF/pytorch-msssim#19]
Hi Author,

You mentioned in pytorch zero-padding is used to replace symmetric padding. 

I think [replication padding](https://pytorch.org/docs/stable/nn.html#torch.nn.ReplicationPad2d) in pytorch is the same as symmetric padding in tf. Would you take a look?
